test/Dialect/Calyx/emit-static.mlir

```diff
@@ -0,0 +1,188 @@
+// RUN: circt-translate --export-calyx --split-input-file --verify-diagnostics %s | FileCheck %s --strict-whitespace
```

Comment:
Can you try round-tripping with the native compiler?

https://docs.calyxir.org/fud/circt.html#native-representation-to-mlir

Reply:
I don't think I can do this in the CIRCT CI currently. As far as I know the Calyx native compiler is not included as an optional dependency of CIRCT. I think there's a decent argument that it should be, but that's beyond the scope of this PR.

Reply:
This can't be done because we don't live in the world of a monorepo. 

So, unfortunately, we need to manually test this. I'm asking if you can ensure that your new changes round trip with the native compiler, i.e., does the native compiler emit correctly emit your new dialect additions. If it is not the case, then please open a bug in the native Calyx GitHub.

Reply:
Oh ok, I understand what you mean now. I will test this manually.

Reply:
Any follow up on this?

Reply:
It doesn't seem to work directly. I will open an issue on the native Calyx repo.